src/components/Paginator/Paginator.tsx

```diff
@@ -0,0 +1,43 @@
+import styles from './Paginator.module.css';
+import { useEffect, useState } from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import { Props } from './types';
+
+export default function Paginator(props: Props) {
+  const { countPages, currentPage } = props;
```

Comment:
you can use destructure inside params: `Pagination({ countPages, currentPage })`